### ISSUE TITLE
[Feature request] dark theme and colors and more

### ISSUE THREAD

**Issue by Rubylion** [dmfs/opentasks#485]
Would love to have it themeable. Also it would be nice to have the opportunity to add pictures to the tasks, showing up on the right side of the list and in the task itself. The progress should be a little more visible. Just suggestions. Keep it up. Great work

**Comment by dmfs** [comment]
We already have #4 for the dark theme. By "pictures" do you mean attachments or these emoji style little icons? We already #57 and #114 for document attachments.

**Comment by Rubylion** [comment]
Thanx, for your replay. I attached a image how i thought. Bigger indicators for importance and more visable progressbars. The images and icons could be visable in the task tab too of course. Maybe you like it. Greetings! 

(https://i.imgur.com/qVtWa81.jpg)

**Comment by dmfs** [comment]
Got it. Since we already have issues for attachments and dark theme, I propose to open separate issues for "additional themes", "preview attachments on list view" and "progress bar visibility"

**Comment by Rubylion** [comment]
On it. Would you consider my suggestions, anyways? Just to know.

**Comment by dmfs** [comment]
Yes, we consider every suggestion. It's more a matter of priorities and order, i.e before we can discuss showing images on the list view, we first need to implement general support for attachments.
